javascript/javascript1/week2/candyhelper.js

```diff
@@ -0,0 +1,44 @@
+const boughtCandyPrices = [];
+const candyTpye = "Sweet";
+function addCandy(candyTpye, weight) {
+  switch (candyTpye) {
+    case "Chocolate":
+      boughtCandyPrices.push(0.7 * weight);
+      break;
+    case "Toffee":
+      boughtCandyPrices.push(1.1 * weight);
+      break;
+    case "Chewing-gum":
+      boughtCandyPrices.push(0.03 * weight);
+      break;
+    default:
+      boughtCandyPrices.push(0.5 * weight);
+  }
+}
+
+// amount to spend
+const amountToSpend = Math.random() * 100;
```

Comment:
This variable returns a number with so many decimals. Please find a way to get rid of decimals or have only two decimals , not more.

Reply:
it can be fixed by using toFixed(2).